### ISSUE TITLE
Timestamp for release requests

### ISSUE THREAD

**Issue by tngraf** [sw360/sw360portal#611]
SW360 should store the the **date** when a user has added ('requested') a release for a projects. Rationale is, that there arer often complains that software clearing takes sooo much time. But in many of these cases a number of releases got requested only a few days ago. Mainline Portal store this date and it was often quite helpful.

**Comment by mcjaeger** [comment]
just a remark, it is in COTS clearing, but in your case: is it required to have it per Group? (I would try to avoid it)

**Comment by tngraf** [comment]
I do not think it is required per Group. This needs to be a property of the project, which is specific per Group anyway. Again: it is not about the time when a release was created, it is about the time when a release was assigned to a project.
Imagine that division X has created a component A and a release A 1.1 somtimes llast year. We want to know WHEN this release got assigned to one of our projects.
